package.json

```diff
@@ -117,13 +117,12 @@
         "rollup-plugin-preserve-shebangs": "^0.2.0",
         "rollup-plugin-styles": "^4.0.0",
         "sloc": "^0.2.1",
-        "storybook": "^7.6.17",
+        "storybook": "^8.4.5",
         "style-loader": "^3.3.3",
         "tiny-invariant": "^1.3.1",
         "typescript": "^5.4.2",
         "typescript-coverage-report": "^0.7.0",
         "vite-plugin-istanbul": "^5.0.0",
-        "vite-plugin-turbosnap": "^1.0.3",
```

Comment:
turbosnap is now built into the "@storybook/react-vite" bundle and can be enabled with the standard --stats-json argument